packages/vaadin-overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -89,6 +89,12 @@ export const PositionMixin = (superClass) =>
       super();
 
       this._updatePosition = this._updatePosition.bind(this);
+
+      this.__targetResizeObserver = new ResizeObserver(() => {
```

Comment:
Could consider using a single `ResizeObserver` instance. See https://github.com/vaadin/web-components/pull/3207#discussion_r773024955

If we want to optimize it further, we could make `overlay-position-mixin` extend `resize-mixin` (would require refactoring it to accept a `resizeTarget` or such)

Reply:
Changed to use a single `ResizeObserver`. We can refactor it later to use `ResizeMixin` if needed.